geth/node/manager_test.go

```diff
@@ -418,21 +479,64 @@ func (s *ManagerTestSuite) TestResetChainData() {
 	FirstBlockHash(require, s.NodeManager, "0x6341fd3daf94b748c72ced5a5b26028f2474f5f00d824504e4fa37a75767e177")
 }
 
-func (s *ManagerTestSuite) TestRestartNode() {
-	require := s.Require()
-	require.NotNil(s.NodeManager)
+func (s *ManagerTestSuite) copyToDir(fromDir string, toDir string) error {
```

Comment:
Do we need this? I saw a comment that we could just start a node with a different directory and no copying is needed.